pkg/apiserver/experiment/experiment.go

```diff
@@ -117,6 +117,14 @@ type SelectorInfo struct {
 	AnnotationSelectors map[string]string `json:"annotation_selectors" binding:"MapSelectorsValid"`
 	FieldSelectors      map[string]string `json:"field_selectors" binding:"MapSelectorsValid"`
 	PhaseSelector       []string          `json:"phase_selectors" binding:"PhaseSelectorsValid"`
+
+	// Pods is a map of string keys and a set values that used to select pods.
+	// The key defines the namespace which pods belong,
+	// and the each values is a set of pod names.
+	Pods map[string][]string `json:"pods" binding:"PodsValid"`
+
+	Mode  string `json:"mode" binding:"oneof='' 'one' 'all' 'fixed' 'fixed' 'fixed-percent' 'random-max-percent'"`
+	Value string `json:"value" binding:"ValueValid"`
```

Comment:
In the current master, `Mode` and `Value` is moved to the field `ScopeInfo`. Please take a look.